jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/ProcessHandler.java

```diff
@@ -454,14 +454,24 @@ private static void linkBoundaryErrorEvent(NodeContainer nodeContainer, Node nod
         ((EventNode)node).setActions(EndNode.EVENT_NODE_EXIT, actions);
     }
     
+    private static String concatName (String prefix, String suffix) {
+        if (prefix == null) {
+            return suffix;
+        } else if (suffix == null) {
+            return prefix;
+        } else {
+            return prefix.concat('-'+suffix);
+        }
+    }
```

Comment:
Just a recommendation for the code style. It should be `concatName(String prefix...)` (no space) and return `prefix.concat('-' + suffix);` (space between operator and operands). If you use Eclipse or IntelliJ IDEA, you can find IDE configuration here: https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration

Reply:
Thanks for the advice. I have code style imported in eclipse but apply it manually because some old files do not follow the format and the result of automatic formatting is messy. In this particular case and the previous one, I forgot to format manually.